Doc/whatsnew/3.10.rst

```diff
@@ -237,6 +243,9 @@ New Features
   :class:`datetime.time` objects.
   (Contributed by Zackery Spytz in :issue:`30155`.)
 
+* Add a :c:func:`PyCodec_Unregister` to unregister a codec search function.
```

Comment:
Please add the word "function" after `PyCodec_Unregister`.

Reply:
updated.